executor/prepared.go

```diff
@@ -262,7 +263,7 @@ func (e *ExecuteExec) Build() error {
 	if err != nil {
 		return errors.Trace(err)
 	}
-	b := newExecutorBuilder(e.Ctx, e.IS)
+	b := newExecutorBuilder(e.Ctx, e.IS, kv.PriorityNormal)
```

Comment:
Yes, we should.
But I don't know how to judge the SQL is  expensive or not here, so just give it a default value.
I think the best way to do it in `plan.Optimize`, use logical plan statement label is too naive. @hanfei1991